lib/runner.js

```diff
@@ -514,8 +502,25 @@ Runner.prototype.hook = function(name, fn) {
       }
       self.emit(constants.EVENT_HOOK_END, hook);
       delete hook.ctx.currentTest;
+      setHookTitle(hook);
```

Comment:
What is this for?

Reply:
When hook.ctx.currentTest is deleted, this changes the hook's title so that it no longer mentions the test case and only mentions the test suite.  It affects the "multiple done calls" errors.